asset/asset.go

```diff
@@ -74,6 +74,16 @@ var (
 )
 
 const (
+	// TaroKeyFamily is the key family used to generate internal keys that
+	// taro will use creating internal taproot keys and also any other keys
+	// used for asset script keys.
+	// This was derived via: sum(map(lambda y: ord(y), 'taro')).
+	// In order words: take the word taro and return the integer
+	// representation of each character and sum those. We get 438, then
+	// divide that by 2, to allow us to fit this into just a 2-byte integer
+	// and to ensure compatibility with the remote signer.
+	TaroKeyFamily = 219
```

Comment:
Lol, that's the second move in a short while :) But makes sense to have it here.